adapters/resetdigital/resetdigital.go

```diff
@@ -0,0 +1,274 @@
+	var (
+		requests []*adapters.RequestData
+		errors   []error
+	)
+
+	referer := getReferer(requestData)
+	currency := getCurrency(requestData)
+
+	if referer == currency {
+		return nil, nil
+	}
+
+	for i := range requestData.Imp {
+		imp := requestData.Imp[i]
+		bidType, err := getBidType(imp)
+
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		splittedRequestData := processDataFromRequest(requestData, imp, bidType)
+
+		requestBody, err := json.Marshal(splittedRequestData)
+		if err != nil {
+			errors = append(errors, err)
+			continue
+		}
+
+		requests = append(requests, &adapters.RequestData{
+			Method:  "POST",
+			Uri:     a.endpointUri,
+			Body:    requestBody,
+			Headers: getHeaders(requestData),
+		})
+	}
+
+	return requests, errors
+}
+
+func processDataFromRequest(requestData *openrtb2.BidRequest, imp openrtb2.Imp, bidType openrtb_ext.BidType) resetDigitalRequest {
+
+	var resetDigitalRequestData resetDigitalRequest
+	resetDigitalRequestData.Site.Domain = requestData.Site.Domain
+	resetDigitalRequestData.Site.Referrer = requestData.Site.Page
+
+	resetDigitalRequestData.Imps = append(resetDigitalRequestData.Imps, resetDigitalRequesImps{})
+	resetDigitalRequestData.Imps[0].BidID = requestData.ID
+	resetDigitalRequestData.Imps[0].ImpID = imp.ID
+
+	var err error
+
+	if bidType == openrtb_ext.BidTypeBanner {
+		resetDigitalRequestData.Imps[0].MediaTypes.Banner.Sizes = append(resetDigitalRequestData.Imps[0].MediaTypes.Banner.Sizes, []int64{imp.Banner.Format[0].W, imp.Banner.Format[0].H})
+	}
+	if bidType == openrtb_ext.BidTypeVideo {
+		resetDigitalRequestData.Imps[0].MediaTypes.Video.Sizes = append(resetDigitalRequestData.Imps[0].MediaTypes.Banner.Sizes, []int64{*imp.Video.W, *imp.Video.H})
+	}
+
+	var extData = make(map[string]interface{})
+	err = json.Unmarshal(imp.Ext, &extData)
+	if err != nil {
+
+	} else {
+
+		resetDigitalRequestData.Imps[0].ZoneID.PlacementID = extData["bidder"].(map[string]interface{})["placement_id"].(string)
+		if resetDigitalRequestData.Imps[0].ZoneID.PlacementID == "test" {
+			resetDigitalRequestData.Imps[0].ForceBid = true
+		}
+
+	}
+	return resetDigitalRequestData
+
+}
+
+func (a *adapter) MakeBids(internalRequest *openrtb2.BidRequest, externalRequest *adapters.RequestData, response *adapters.ResponseData) (*adapters.BidderResponse, []error) {
+	if response.StatusCode == http.StatusNoContent {
+		return nil, nil
+	}
+	if response.StatusCode == http.StatusBadRequest {
+		return nil, nil
+	}
+	if response.StatusCode != http.StatusOK {
+		return nil, nil
+	}
+
+	if err := json.Unmarshal(response.Body, &response); err != nil {
+		return nil, []error{err}
+	}
+	bidResponse := adapters.NewBidderResponseWithBidsCapacity(MaxBids)
+	//check no bids
+	jsonData := make(map[string]interface{})
+
+	json.Unmarshal([]byte(response.Body), &jsonData)
+	//Always one bid
+	bid := getBidFromResponse(jsonData)
```

Comment:
@bruno-siira bidder docs PR should explicity specify the behaviour of considering one bid from  `jsonData`